openhtf/exe/__init__.py

```diff
@@ -36,6 +36,11 @@ class TestStopError(Exception):
   """Test is being stopped."""
 
 
+def _InvalidTestStart():
```

Comment:
What do we do for the serial if it's None?

On Fri, Apr 22, 2016 4:05 PM, grybmadsci notifications@github.com wrote:
In openhtf/exe/__init__.py :

> @@ -36,6 +36,11 @@ class TestStopError(Exception):
>    """Test is being stopped."""
>  
>  
> +def _InvalidTestStart():


Hm, I'm not sure we need to enforce this. test_start is more of a 'wait for this
before starting', so None implies "don't wait for anything before starting",
which seems reasonable to me. AutoStart has been deprecated for a similar reason
- starting the test automatically is the default not the exceptional case. Can
we just check for None in TestExecutor._WaitForTestStart() and return immediately in that case?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub